### PR TITLE
Ndev 1837 solana compasibility

### PR DESCRIPTION
**Pull request by kristinaNikolaevaa**
Can be merged only after https://github.com/neonlabsorg/neon-evm/pull/272

### CODE REVIEW

File: integration/tests/economy/test_economics.py [hunk 1]
```diff
@@ -328,7 +328,8 @@ def test_deploy_small_contract_less_100tx(
 
         sol_balance_after_deploy = operator.get_solana_balance()
         token_balance_after_deploy = operator.get_token_balance(w3_client)
-        tx = w3_client.make_raw_tx(account_with_all_tokens.address)
+        tx = w3_client.make_raw_tx(account_with_all_tokens.address, gas=0)
```
Comment (gigimon) [inline integration/tests/economy/test_economics.py]: Why did you add gas=0?

File: integration/tests/neon_evm/test_interoperability.py [hunk 2]
```diff
@@ -1,95 +1,100 @@
...
-        evm_loader,
-        treasury_pool,
-        sol_client,
-    ):
+    def test_get_solana_address_by_neon_address(self, sender_with_tokens, solana_caller):
+        sol_addr = solana_caller.get_solana_address_by_neon_address(sender_with_tokens.eth_address.hex())
+        assert sol_addr == sender_with_tokens.solana_account_address
+
+    def test_get_payer(self, solana_caller):
+        assert solana_caller.get_payer() != ""
```
Comment (gigimon) [inline integration/tests/neon_evm/test_interoperability.py:71]: Check the payer address. Is it Solana (or eth?) address and it exist?

File: integration/tests/neon_evm/test_interoperability.py [hunk 3]
```diff
@@ -1,95 +1,100 @@
...
+
+    def test_get_eth_ext_authority(self, solana_caller, sender_with_tokens):
+        addr = solana_caller.get_eth_ext_authority(b"123", sender_with_tokens)
+        assert addr != ""
+
+    def test_create_resource(self, sender_with_tokens, solana_caller):
+        salt = b"123"
+        resource_address = solana_caller.create_resource(sender_with_tokens, salt, 8, 1000000000, MEMO_PROGRAM_ID)
+        acc_info = solana_client.get_account_info(resource_address, commitment=Confirmed)
+        assert acc_info.value is not None
```
Comment (gigimon) [inline integration/tests/neon_evm/test_interoperability.py:86]: check size for account and balance (to prevent remove it)

File: integration/tests/neon_evm/utils/instructions.py [hunk 4]
```diff
@@ -135,7 +136,12 @@ def make_ExecuteTrxFromAccount(
         accounts.append(
             AccountMeta(acc, is_signer=False, is_writable=True),
         )
-
+    if additional_signers:
+        for acc in additional_signers:
+            print("Additional acc ", acc.public_key)
```
Comment (gigimon) [inline integration/tests/neon_evm/utils/instructions.py]: print

**Review by gigimon — APPROVED** [review]

File: integration/tests/economy/test_economics.py (hunk 1 above)
Comment (kristinaNikolaevaa) [reply integration/tests/economy/test_economics.py]: I didn't
it is changes from develop

File: integration/tests/neon_evm/test_interoperability.py (hunk 2 above)
Comment (kristinaNikolaevaa) [reply integration/tests/neon_evm/test_interoperability.py:71]: No, it is not exist. I added the check after using payer in some transaction that there are no lamports and data after using (test test_execute_from_account_create_acc)
